src/pages/paywall/article/[id]/index.tsx

```diff
@@ -1,63 +1,85 @@
 import { useRouter } from 'next/router';
 import { Skeleton, SkeletonTypeMap } from '@mui/material';
-import Typography from '@mui/material/Typography';
-import Alert from '@mui/material/Alert';
-import { useGetArticle } from '../../../../client/api/personalization/use-get-article';
 import { UseCaseWrapper } from '../../../../client/components/common/UseCaseWrapper/UseCaseWrapper';
 import { CustomPageProps } from '../../../_app';
+import { USE_CASES } from '../../../../client/components/common/content';
+import LinkArrow from '../../../../client/img/externalLinkArrow.svg';
 import Link from 'next/link';
-import { ArrowBack } from '@mui/icons-material';
+import Image from 'next/image';
+import styles from '../../paywall.module.scss';
+import Alert from '../../../../client/components/common/Alert/Alert';
+import { ArticleGrid, Byline } from '../..';
```

Comment:
It looks extremely strange! It would be cool to move `ArticleGrid` and `Byline` components to a separate file.

Reply:
You are right, I have moved the common card and card grid components into a separate file.